### PR TITLE
Use raw JSON instead of ArchivesSnake JSONModel objects

### LINKED ISSUE #91: Modify code to work with raw JSON instead of JSON model objects
**Is your feature request related to a problem? Please describe.**
Functions currently expect JSON model objects instead of raw JSON. In order for code in this project to be incorporated into the ArchivesSnake library, functions and decorators should be modified to expect raw JSON.

**Describe the solution you'd like**
Code should be modified to work with raw JSON instead of JSON model objects. Decorators may need to be adjusted as well.

**Additional context**
Tests will need to be updated; this will be covered by a separate issue.

### PR DESCRIPTION
**Pull request by p-galligan**
## Changes
Fixes #91. Also bundles in issue 92 and 84. 

## Blockers
Future direction for instantiating ASnake client for functions that requiring interacting with ASnake and how to update those for raw JSON use. Will have to reach out to Dave Mayo for guidance.

### CODE REVIEW

File: rac_aspace/data_helpers.py [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
 
 
-@check_type(JSONModelObject)
+@check_type(dict)
 def get_note_text(note):
     """Parses note content from different note types.
 
     Args:
-        note (JSONModelObject): an ArchivesSpace note object.
+        note (array): an ArchivesSpace note.
```
Comment (helrond) [inline rac_aspace/data_helpers.py:23]: Err, shouldn't this be a dict?

File: rac_aspace/data_helpers.py [hunk 2]
```diff
@@ -29,45 +29,46 @@ def parse_subnote(subnote):
         """Parses note content from subnotes.
 
         Args:
-            subnote (JSONModelObject): an ArchivesSpace subnote object.
+            subnote (array): an ArchivesSpace subnote.
```
Comment (helrond) [inline rac_aspace/data_helpers.py:32]: Again, I think this is a dict, not an array?

**Review by helrond — COMMENTED** [review]
I started leaving comments, but then realized I was leaving comments on docstrings, which is work that others are taking on this sprint. So feel free to ignore those.

My only other comment is that there's a mix of single quotes and double quotes in this file. I'd suggest we consistently use double quotes throughout - it makes reading this code a little less distracting and improves maintainability.

**Review by helrond — APPROVED** [review]

**Comment by HaSistrunk** [comment]
Re: @helrond 's comments on docstrings: Issue #90 is to change docstrings style from Google to Sphinx. If we want to do a deeper edit of the code documentation to fix inaccuracies, than perhaps we should add another issue (or we can expand #90)? We were assuming the existing docstrings were correct in tackling issue #90.

**Review by HaSistrunk — APPROVED** [review]

**Comment by p-galligan** [comment]
@helrond @HaSistrunk I'm happy to make content changes here if you want, we'd just need to make that decision before merging.

**Comment by HaSistrunk** [comment]
I'm cool to merge this, and tackle all docstring changes in #90 .

**Comment by p-galligan** [comment]
Fixes #91

**Pull request merged**